doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -368,6 +368,13 @@ Libraries for networking
   * Added :c:func:`nrf_cloud_rest_send_device_message` function that sends a JSON message to nRF Cloud using the `SendDeviceMessage` endpoint.
   * Added :c:func:`nrf_cloud_rest_send_location` function that sends an NMEA sentence to nRF Cloud as a GPS-type device message.
 
+Libraries for NFC
+-----------------
+
+* :ref:`nfc_ndef_parser_readme`
+  * Added an assertion in the :c:func:`nfc_ndef_msg_parse` to avoid a potential usage fault and added a note in the function documentation.
+  * Renamed the :c:macro:`NFC_NDEF_PARSER_REQUIRED_MEM`.
```

Comment:
```suggestion
* :ref:`nfc_ndef_parser_readme`

  * Added an assertion in the :c:func:`nfc_ndef_msg_parse` to avoid a potential usage fault and added a note in the function documentation.
  * Renamed the :c:macro:`NFC_NDEF_PARSER_REQUIRED_MEM`.
```

Reply:
Fixed, but also added further modification and reformatted this statement.